tests/bsim/bluetooth/audio/src/cap_commander_test.c

```diff
@@ -549,6 +1078,60 @@ static void test_change_microphone_gain(void)
 	printk("Microphone gain changed\n");
 }
 
+static void test_broadcast_reception_start(size_t accepter_count)
+{
+	struct bt_cap_commander_broadcast_reception_start_param reception_start_param;
+	struct bt_cap_commander_broadcast_reception_start_member_param param[CONFIG_BT_MAX_CONN];
```

Comment:
Suggest to either memset these or initialize them to `0` to avoid any future uninitialized values